packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -285,11 +294,10 @@ export const CalendarMonth = ({
                   onMonthChange(ev, newDate);
                 }, 0);
               }}
-              variant="single"
-              selections={monthFormatted}
+              selected={monthFormatted}
             >
               {longMonths.map((longMonth, index) => (
-                <SelectOption key={index} value={index} isSelected={longMonth === monthFormatted}>
+                <SelectOption key={index} itemId={index} isSelected={longMonth === monthFormatted}>
                   {longMonth}
                 </SelectOption>
               ))}
```

Comment:
Just need to wrap this in the SelectList sub-component. Right now it's rendering the `li`'s without any list element.

This will also apply to the other example/demo files updates to use the new Select